src/components/DateRelated.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import colors from '@styles/colors';
+import { format } from 'date-fns';
+import { ru } from 'date-fns/locale';
+import { WeatherIcon, WeatherType } from '@components/WeatherIcon';
+
+interface IProps {
+    dayOfTheWeek: string;
+    date: string;
+    weatherType: WeatherType;
+    temperatureAfternoon: number;
+    temperatureNight: number;
+}
+
+const getWeek = format(new Date(), 'cccc', { locale: ru });
```

Comment:
make it return values not only for current date, so we could get weekday from date prop